WalletWasabi.Fluent/Views/NavBar/NavBar.axaml

```diff
@@ -94,7 +94,11 @@
   </UserControl.Styles>
   <DockPanel Margin="0 0 0 4">
     <StackPanel DockPanel.Dock="Top" Spacing="5" Background="Transparent">
```

Comment:
Remove Background="Transparent"

Reply:
If I remember right that was needed to get the `IsPointerOver` value correctly. For switching logo to hamburger icon when the cursor enters to the NavBar's area.